notebooks/official/tensorboard/tensorboard_custom_training_with_custom_container.ipynb

```diff
@@ -0,0 +1,999 @@
+{
```

Comment:
<p>Update the links to the real location of this notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I think these links are not working because I have not submitted the notebook. I copied the URL patten from other submitted notebooks so that links should work after we submit this RP. Is my understanding correct?</p>

Reply:
<p>Sorry did not noticed the diff between community and official. Updated.</p>